src/components/search/SearchBlock.js

```diff
@@ -46,12 +46,21 @@ export default function SearchBlock({
     startTransition(async () => {
       if (text?.length > 1) {
         const users = await usersService.search({
-          login: text,
+          keyword: text,
           limit: 5,
         });
-        setSearchedUsers(users);
+
+        const sortedUsers = users.sort((a, b) => {
+          const lenDiff = a.login.length - b.login.length;
```

Comment:
maybe we could provide a `sort` param for search query, so the backend will do this for us?
WDYT?

Reply:
We already have server-side sorting, but it works quite strangely and does not sort by length

server-side param: `{ first_name: -1, last_name: 1, login: 1 }`

So I thought it would be better to separate this logic to the client side

Reply:
aah ok, sorry, I did not notice it sorts by `length`, thought it was `login`